test/dbus/lvmdbustest.py

```diff
@@ -465,6 +463,11 @@ def tearDown(self):
 					# print('Re-creating PV=', p)
 					self._pv_create(p)
 
+	def tearDown(self):
+		# If we get here it means we passed setUp, so lets remove anything
+		# and everything that remains, besides the PVs themselves
+		self.clean_up()
```

Comment:
[addCleanup](https://docs.python.org/3/library/unittest.html#unittest.TestCase.addCleanup) might be useful here, we use it in our tests because `tearDown` doesn't run if `setUp`.

Reply:
Thanks, added